### DISCUSSION TITLE
Off route alarm.

### DISCUSSION THREAD

**Discussion by gammeredith** [osmandapp/OsmAnd#11123] [Q&A]
Re: Bicycle profile. 

I am no longer receiving the voice instruction 
"You have been off the route for 50 metres" 
Voice announcements are set to normal. I am using using Osmands native navigation feature. 
Am I missing a setting here.

**Comment by vshcherb** [comment]
Please check Voice instructions - Announce distance and it should be listed there with Information at what distance it will be triggered.

**Reply by Lee-Carre** [reply]
Might be related to #12672.
